extensions/jaxb/deployment/src/main/java/org/apache/camel/quarkus/component/jaxb/deployment/JaxbProcessor.java

```diff
@@ -35,4 +39,10 @@ RuntimeInitializedClassBuildItem fastInfosetStreamWriterOutput() {
         return new RuntimeInitializedClassBuildItem(FastInfosetStreamWriterOutput.class.getCanonicalName());
     }
 
+    @BuildStep
+    @Record(value = ExecutionTime.STATIC_INIT)
+    CamelContextCustomizerBuildItem fastInfosetStreamWriterOutput(JaxbRecorder recorder) {
```

Comment:
Is this a correct method name - fast info set?

Reply:
Of course not. My copy & paste mistake. Thanks, let me fix it.

Reply:
Fixed in 8f9a698